config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2.1
```

Comment:
If this is a CircleCI config.yml file, why is it not in the .circle directory? (Well, there's already one there in fact -- so is this a duplicate?)

Reply:
Yeah, this was a duplicate, just removed it.